src/tests/HDFFile.cxx

```diff
@@ -1265,3 +1255,43 @@ TEST_F(T_CommandHandler, data_f142) { T_CommandHandler::data_f142(); }
 TEST_F(T_CommandHandler, dataset_static_1d_string_variable) {
   T_CommandHandler::dataset_static_1d_string_variable();
 }
+
+template <typename T>
+void testType(FileWriter::HDFFile &TestFile,
```

Comment:
Would a function name like `verifyWrittenDatatype` be more appropriate? Because, all datatypes are are written to the same HDF file in the same test run.